### PR TITLE
slider offset

### PR DESCRIPTION
**Pull request by dominik-git**

### CODE REVIEW

File: src/components/series-detail/series-container.css [hunk 1]
```diff
@@ -8,7 +8,7 @@
 }
 
 .series-content {
-  margin-left: 100px;
+  margin-left: 115px;
```
Comment (MarekTalpas) [inline src/components/series-detail/series-container.css]: you forgot to assign it into variable, like that - var(--home-container--margin-left)

Comment (dominik-git) [inline src/components/series-detail/series-container.css]: it was not in variable before.

Comment (MarekTalpas) [inline src/components/series-detail/series-container.css]: but now it will be in variable.

Comment (dominik-git) [inline src/components/series-detail/series-container.css]: it won't

**Pull request merged**